ce/ce/cli/switches/dump-env.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { i } from '../../i18n';
+import { session } from '../../main';
+import { Uri } from '../../util/uri';
+import { resolvePath } from '../command-line';
+import { Switch } from '../switch';
+
+export class DumpEnv extends Switch {
+  
+  switch = 'dump-env';

```

Comment:
I'd probably have gone with something like `--json` for this so it's not specifically saying 'env' (since it's also got other definitions in there)